site/docs/contribute/build-test.md

```diff
@@ -62,11 +62,18 @@ export DOCKER_NAMESPACE=m4d-system
 # build a local kind cluser
 make kind
 
-# deploy the the cluster 3rd party such as cert-manager and vault
-make cluster-prepare
+# deploy vault
+make -C charts vault
+make -C charts wait-for-vault
 
-# build all docker images and push them to the local registry
-make docker
+# check if cert-manager is already installed
+kubectl get namespaces
+
+# If cert-manager namespace did not appear in the list from the previous step, install cert-manager
+make -C charts cert-manager
+
+# Deploy the datashim which enables dynamic bucket allocation in object store
+make -C third_party/datashim deploy
 
 # build the mock/test docker images and push them to local registry
 make -C test/services docker-build docker-push
```

Comment:
`make -C test/dummy-mover docker-build docker-push` is  missing. Without this image, run-integration-tests fails.

Reply:
@shlomitk1 Added.  Thanks!